posthog/hogql_queries/insights/trends/trends_query_runner.py

```diff
@@ -184,7 +184,7 @@ def run(index: int, query: ast.SelectQuery | ast.SelectUnionQuery, is_parallel:
 
         # This exists so that we're not spawning threads during unit tests. We can't do
         # this right now due to the lack of multithreaded support of Django
-        if settings.IN_UNIT_TESTING:  # type: ignore
+        if settings.IN_UNIT_TESTING:
```

Comment:
Same as in `posthog/celery.py`.

Reply:
Nevermind, the solution involved manually updating the `mypy_baseline.txt` file.